### PR TITLE
Fix mode type flags

### PR DESCRIPTION
**Pull request by tomstokes**
Mode type is in mode.type_, not mode.flags

### CODE REVIEW

**Comment by tomstokes** [comment]
Thanks for adding bitflags. Makes the API much easier to work with.

However, the mode type comes from `drm_display_mode.type`, not the `.flags` field. Easy fix.

**Comment by MarijnS95** [comment]
Whoops! Looks like it was already introduced like that in 2467a9e232240200c468b381e691d0f4b36b4de0 though, `ModeType` type with `DRM_MODE_TYPE_*` bitmasks and all.

Should we also add a `bitflags!` type and getter for `DRM_MODE_FLAG_*` then, representing this `.flags` field?

**Comment by tomstokes** [comment]
> Should we also add a `bitflags!` type and getter for `DRM_MODE_FLAG_*` then, representing this `.flags` field?

I could add that, too.

Would you prefer it added to this PR, or should I open a separate PR for the `.flags` getter and types?

**Comment by MarijnS95** [comment]
I'm not the maintainer, but I'd prefer to do that in a separate PR as to unblock this bugfix and get it in ASAP.

@tomstokes It looks like `crates.io` had a little hiccup (also saw that on other repositories), would you mind restarting the CI so that it's green when @Drakulix comes along to merge this?

**Review by MarijnS95 — APPROVED** [review]

**Comment by Drakulix** [comment]
> > Should we also add a `bitflags!` type and getter for `DRM_MODE_FLAG_*` then, representing this `.flags` field?
> 
> I could add that, too.
> 
> Would you prefer it added to this PR, or should I open a separate PR for the `.flags` getter and types?

Please do another PR, when you got the time!

Thanks for this

**Pull request merged**
